sample_command_policy.json

```diff
@@ -0,0 +1,28 @@
+{
+  "commands" : [
+    [ 
+        "cd examples/c_code_proj;  ./makefile",
+        "ls -ltr examples/c_code_proj/test",
+        "build_snap",
+        "default_policy.json"
+    ],
+    [
+        "cd examples/java_maven_proj;  mvn clean",
+        "ls -ltr examples/java_maven_proj/target",
+        "build_main",
+        "README.md"
+    ],
+    [
+        "cd tuf; sudo python setup.py install",
+        null,
+        "build_tuf_install",
+        null
+    ],
+    [
+        "cd examples/c_code_proj;  ./makefile_err",
+        "ls -ltr testErr",
+        "build_snapErr",
+        null
+    ]
+  ]
+}
```

Comment:
The format here is somewhat opaque, in that when reading this policy file, it's not immediately clear what each of these entries corresponds to (e.g. input vs. cmd vs. cmd_name). Perhaps this should take a more traditional JSON format where each sublist is instead a "sub-dictionary"?